added gcloud-components-update.md to common (#11682)
tldr-pages/tldr: pages/common/gcloud-components-update.md

# gcloud components update

> > Install specific components of the Google Cloud CLI, along with their dependencies.
> > Installs components at the current version of the Google Cloud CLI without upgrading the existing installation.

Reviewer: ```suggestion
> Update all your installed Google Cloud CLI components to the latest version.
> See also: `gcloud`.
```

> More information: <https://cloud.google.com/sdk/gcloud/reference/components/update>.

- Update all components to the latest version:

`gcloud components update`

> - Update all components to a specific version (e.g., 1.2.3):
> 
> `gcloud components update --version=1.2.3`

Reviewer: ```suggestion
- Update all components to a specific version:

`gcloud components update --version={{1.2.3}}`
```

We can update this to a placeholder instead, to be highlighted by our clients.

> - Check the current version of Google Cloud CLI:
> 
> `gcloud version`
> 
> - List all available components and their versions:
> 
> `gcloud components list`
> 

Reviewer 2: The page documents `gcloud components update`, but these examples are for `list` and `version`.
```suggestion
```

Author: Thanks, I appreciate the help
